packages/react-widgets/__tests__/models/utils.test.js

```diff
@@ -77,7 +77,7 @@ describe('utils', () => {
       };
       const query = formatTableNameWithFilters({ source });
 
-      expect(query).toBe(`(${source.data.replace(';', '')})`);
+      expect(query).toBe(`(SELECT * FROM test) foo`);
```

Comment:
Probably 'foo' here is not the best name... @Clebal, don't you think? `queryAlias`, as in the PR title would be better

Reply:
queryAlias is too long. foo is a classic label for something that isn't useful like in this case. I would even use only `q`.

Reply:
I know... 😄 
![image](https://user-images.githubusercontent.com/458196/165998423-bcfc2547-1cfb-4028-bc8d-99b3e533972a.png)
https://www.genbeta.com/desarrollo/rfc-3092-el-curioso-origen-de-las-variables-foo-y-bar

Reply:
Now that I know that, I like it more hahaha.